### ISSUE TITLE
Feature request: Extension - Rightclick - Copy as text

### ISSUE THREAD

**Issue by martinandersen3d** [Microsoft/vscode#47296]
I often do **documentation on extensions** i install.

it would be nice if I could rightclick on a extension in the Extension panel, and copy this text:

**Example of the copied text:**

Extension: Stylelint
Description: Modern CSS/SCSS/Less linter
Publisher:"Shinnosuke Watanabe"
Github/ MS-Marketplace:  https://marketplace.visualstudio.com/items?itemName=shinnn.stylelint

-----
Any idears for improvements?

**Comment by roblourens** [comment]
The `code --list-extensions` CLI command might be helpful.

**Comment by sandy081** [comment]
@martinandersen3d You can get all installed extensions using above option. Let me know if it is enough or .

**Comment by lannonbr** [comment]
So I did some preliminary work on this. I added a new IExtensionAction to the cog on the extension item which takes the data as described in the first post of this thread and write it to the clipboard. Afterwards, I push a small notification to the screen saying the info was saved to the clipboard. I will have to add some stuff for i18n purposes, but otherwise I am fairly close to making a PR for this.
